### PR TITLE
Create B827EBFFFEFB5FBC.json

### PR DESCRIPTION
**Pull request by hsuarez1215**
We appreciate your help with the creation of the file.

### CODE REVIEW

**Comment by hsuarez1215** [comment]
This is my first LoraWAN gateway, best regards

File: B827EBFFFEFB5FBC.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+		"gateway_ID": "B827EBFFFEFB5FBC", 
+		"servers": [ 
+			{ 
+				"server_address": "router.us.thethings.network", 
+				"serv_port_up": 1700, 
+				"serv_port_down": 1700, 
+				"serv_enabled": true 
+			} 
+		], 
+		"ref_latitude": 0,
```
Comment (gonzalocasas) [inline B827EBFFFEFB5FBC.json]: Please enter at least a rough location. Thx!

**Comment by hsuarez1215** [comment]
I have already updated the configuration file.

**Comment by hsuarez1215** [comment]
Do I have any other error in my file?
I searched the name of the file in the repository but it has not yet been created.

**Comment by gonzalocasas** [comment]
yes, there was a missing opening curly brace, but I just fixed and merged it :)

**Pull request merged**